ocs_ci/deployment/deployment.py

```diff
@@ -820,6 +820,8 @@ def get_device_paths(worker_names):
     platform = config.ENV_DATA.get('platform').lower()
     if platform == 'aws':
         pattern = 'nvme-Amazon_EC2_NVMe_Instance_Storage'
+    elif platform == 'none'
```

Comment:
We should use a more explicit `platform` value for baremetal, like `'baremetal'` or `'bm'`. I know baremetal isn't really a "platform" but it will be clearer to users that this is baremetal and not aws, vsphere, etc.

Reply:
@clacroix12 As of now, there's no platform identifier written for baremetal unlike `AWS` and `vSphere`. When a BMC is deployed, the cluster's console shows `none`  as of today. But it will be getting changed in the near future to somethinglike `baremetal`/`bm`

Reply:
@rajatsingh25aug I understand, but this is something we set in our own configuration so it's independent of the console. If we can set `baremetal / bm` as the platform in our config and key off of it here it will be clearer to those maintaining this code.

Reply:
Ah, I was thinking that we have to make changes by keeping the UI in the mind. Then`baremetal` sounds logical.
cc @anubhav-here

Reply:
@clacroix12  please set it as `baremetal` .

Reply:
Changed it to ```baremetal```

Reply:
@rajatsingh25aug @anubhav-here I don't think we have a baremetal config in master yet. I assumed this would be something that is necessary for baremetal deployments / this PR. 

If the intention here is is that the baremetal deployment is done separately and we are only executing tests through ocs-ci we can either revert back to using `'none'` for now or you can create a new config with this:

```yaml
---
ENV_DATA:
  platform: 'baremetal'
```

This will obviously need to be passed to ocs-ci when executing tests against a baremetal cluster.

Reply:
@clacroix12 Cool. I changed it back to ```none``` for now as there is no config associated with ```baremetal``` as of now.